lxd/instance/drivers/driver_qemu.go

```diff
@@ -24,7 +24,9 @@ import (
 	"sort"
 	"strconv"
 	"strings"
+	"syscall"
```

Comment:
Please use `golang.org/x/sys/unix` as `syscall` is deprecated.

Reply:
Thanks, I remember another PR a few weeks back... :)

Reply:
Switched to `os.O_RDWR`.